tutorials/basic-topics/sorting/insertion-sort.md

```diff
@@ -11,4 +11,222 @@ keywords:
   - insertion sort
 ---
 
-<TutorialAuthors names="@TBC"/>
+<TutorialAuthors names="@Shivashish-rwt"/>
+
+## Overview
+
+Insertion sort is one of the sorting algorithms that sort the elements by placing an unsorted element in correct sorted order one at a time.
+
+First of all, let us understand how an sorted `array` (If you don't what an array is. Please refer [here](/tutorials/basic-topics/arrays.md)) looks like. Understanding an sorted array will give us the overview of the insertion sort. An sorted array is given below.
+
+Example: [1,2,3,4,5,6,7]
+
+In a sorted `array` all the elements to the left any element are smaller than that. That, means to make the `array` sorted we have to place an element in such a position, so that every element to the left are smaller than that. That is pretty much what we do in a insertion sort.
```

Comment:
What does this mean ? Correct the sentence properly
In a sorted `array` all the elements to the left any element are smaller than that.

Reply:
Check this:

![sentence](https://user-images.githubusercontent.com/85662806/194008777-052c233b-755f-4454-9d80-23cc5c74352e.png)

Reply:
@vigneshshiv 
Could u please check if the above mentioned changes are correct or not so that i can work on that.

Reply:
We can correct this, leave it.